tests/test.py

```diff
@@ -521,6 +522,14 @@ def test_export(self):
 
         self.assertEqual(exported_values, value_list)
 
+    def test_get_gid(self):
+        self.sheet._id = 'od6'
+        self.assertEqual(self.gid, '0')
```

Comment:
`self` refers to `WorksheetTest` instance, so this results in `AttributeError: 'WorksheetTest' object has no attribute 'gid'`.